### DISCUSSION TITLE
[Feature request]  adding captions to audio books

### DISCUSSION THREAD

**Discussion by bstacks544** [PaulWoitaschek/Voice#1403] [Ideas]
Please include the following details in your bug report:

App-Version:
Phone Name:
Android Version:

**Comment by maribox** [comment]
This could be extended to live captions while listening to books, so I can read through automatically generated text

**Comment by cyruseuros** [comment]
I think the first step would be supporting .srt files. Those can be generated fairly easily by end users these days. After that it might make sense to look into generating them on the fly
